public/video-ui/styles/components/_buttons.scss

```diff
@@ -74,7 +74,7 @@ button {
 
 .btn--loading {
   background: {
-    image: url(/assets/video-ui/images/progress-spinner.gif);
+    image: url(../images/progress-spinner.gif);
```

Comment:
Can't find the docs on it now, but upgrading webpack resulted in needing to change these to relative imports rather than absolute.